docker-compose.yaml

```
@@ -3,24 +3,28 @@ version: "3.8"
 services:
   search:
     build:
+      context: .
       target: search-image
     logging:
       driver: "json-file"
       options:
         max-size: "10m"
         max-file: "5"
     ports:
-      - "127.0.0.1:5064:5064"
+      - "5064:5064"
     environment:
       - HOST=0.0.0.0
       - PORT=5064
+      - ELASTICSEARCH_URI=elasticsearch
```

Please specify explicitly the port here:
```suggestion
      - ELASTICSEARCH_URI=elasticsearch:9200
```